contrib_bots/lib/github.md

```diff
@@ -0,0 +1,68 @@
+# Zulip Github API
+## Getting Started
+1. Add `from . import github` to the top of your script.
+2. Initialise the class by adding `gh = github()`
+3. Authenticate the session with `gh.auth()`
+4. Use any features you want to use. Visit [The developer Website](https://developer.github.com/v3)
+5. Make a .github folder in your home directory, and make a github.conf file in it
```

Comment:
I think all the steps about setting up configuration should actually come first here.